packages/ffe-datepicker-react/src/datepicker/SpinButton.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef } from 'react';
+import classNames from 'classnames';
+
+interface SpinButtonProps extends React.ComponentPropsWithoutRef<'span'> {
+    className?: string;
+    onSpinButtonChange: (value: number[], allowFocusNext?: boolean) => void;
+    children: React.ReactNode;
+    maxLength: number;
+    min: number;
+    max: number;
+    value?: number;
+    nextSpinButton?: React.RefObject<HTMLSpanElement>;
+    prevSpinButton?: React.RefObject<HTMLSpanElement>;
+}
+
+export const SpinButton = React.forwardRef<HTMLSpanElement, SpinButtonProps>(
+    (
+        {
+            className,
+            onSpinButtonChange,
+            maxLength,
+            min,
+            max,
+            value,
+            nextSpinButton,
+            prevSpinButton,
+            children,
+            ...rest
+        },
+        ref,
+    ) => {
+        const history = useRef<number[]>([]);
+
+        const handleKeyDown = (evt: React.KeyboardEvent) => {
+            evt.stopPropagation();
+
+            if (/\d/.test(evt.key)) {
+                history.current =
+                    history.current.length === maxLength
+                        ? (history.current = [parseInt(evt.key)])
+                        : history.current.concat(parseInt(evt.key));
+                onSpinButtonChange(history.current);
+            } else if (evt.key === 'Backspace') {
+                history.current = [];
+                onSpinButtonChange(history.current);
+            } else if (evt.key === 'ArrowUp') {
+                let newValue = (value ?? 0) + 1;
+                if (newValue && newValue !== null && newValue > max) {
+                    newValue = min;
+                }
+                onSpinButtonChange([newValue], false);
+            } else if (evt.key === 'ArrowDown') {
+                let newValue = (value ?? 0) - 1;
+                if (newValue < min) {
+                    newValue = max;
+                }
+                onSpinButtonChange([newValue], false);
+            } else if (evt.key === 'ArrowLeft') {
+                if (prevSpinButton) {
```

Comment:
```suggestion
               prevSpinButton.current?.focus();
```

if trengs ikke eftersom du bruker `?.`. 

Jag synes kanskje det er ryddigare att focus logiken utanfør.